app.py

```diff
@@ -1,2 +1,46 @@
-from flask import Flask, request
+from flask import Flask, request, jsonify
 import json
```

Comment:
small comment: 
it is a good practice to order the inputs in this way:
1. standard library packages
2. installed packages
3. your own packages

e.g.
```python
import json

from flask import Flask, request, jsonify

import mymodule
```

Reply:
fixed